hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/security/OzoneDelegationTokenSelector.java

```diff
@@ -44,7 +44,9 @@ public OzoneDelegationTokenSelector() {
   public Token selectToken(Text service,
       Collection<Token<? extends TokenIdentifier>> tokens) {
     LOG.trace("Getting token for service {}", service);
-    return super.selectToken(service, tokens);
+    Token token = super.selectToken(service, tokens);
+    LOG.info("Got tokens: {} for service {}", token, service);
```

Comment:
shall we dial it down to debug level?

Reply:
Yes, I will definitely change this to DEBUG.